hedera-node/hedera-app/src/main/java/module-info.java

```diff
@@ -53,10 +53,13 @@
     requires io.netty.transport.classes.epoll;
     requires io.netty.transport;
     requires java.annotation;
+    requires jdk.jdi;
     requires org.apache.commons.lang3;
     requires static com.github.spotbugs.annotations;
     requires static com.google.auto.service;
-    requires static java.compiler; // javax.annotation.processing.Generated
+    requires static java.compiler;
+    requires static org.jetbrains.annotations;
```

Comment:
Is it intentional to keep `org.jetbrains.annotations` in this PR?

Reply:
No, was a typo. Removed. Thanks @andrewb1269hg